Jenkinsfile

```diff
@@ -0,0 +1,101 @@
+#!groovy
+
+def GIT_BRANCH = ''
+def IMAGE_REF = ''
+def IMAGE_TAG = ''
+def NOTIFIER_IMAGE = 'argo-rest-notifier'
+def VERSION = ''
+def NAMESPACE = ''
+
+def runUtilityCommand(buildCommand) {
+    // Run an arbitrary command inside the docker builder image
+    sh "docker run --rm " +
+       "-v ${pwd()}/dist/pkg:/root/go/pkg " +
+       "-v ${pwd()}:/root/go/src/github.com/argoproj/argo " +
+       "-w /root/go/src/github.com/argoproj/argo argo-builder ${buildCommand}"
+}
+
+def dockerPush(dockerImageTag) {
```

Comment:
delete this and use https://github.com/CyrusBiotechnology/jenkins-pipeline-library/pull/14/files